apps/web/pages/cancel/[uid].tsx

```diff
@@ -98,29 +100,30 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
                               : t("cancelling_event_recurring")}
                           </p>
                         </div>
-                        <div className="mt-4 border-t border-b py-4">
-                          <h2 className="font-cal mb-2 text-center text-lg font-medium text-gray-600">
-                            {props.booking?.title}
-                          </h2>
-                          {props.booking?.eventType.recurringEvent &&
-                            props.booking?.eventType.recurringEvent.freq &&
-                            props.recurringInstances && (
-                              <div className="text-center text-gray-500">
-                                <Icon.RefreshCcw className="mr-3 -mt-1 ml-[2px] inline-block h-4 w-4 text-gray-400" />
-                                <p className="mb-1 -ml-2 inline px-2 py-1">
-                                  {getEveryFreqFor({
-                                    t,
-                                    recurringEvent: props.booking.eventType.recurringEvent,
-                                    recurringCount: props.recurringInstances.length,
-                                  })}
-                                </p>
-                              </div>
-                            )}
-                          <div className="text-gray-500">
-                            <div className="flex flex-row items-start justify-center space-x-3">
+                        <div className="ptext-left mt-8 grid grid-cols-3 border-t border-b py-8">
+                          <h2 className="font-medium">{t("event_name")}</h2>
+                          <h2 className="col-span-2 mb-12">{props.booking?.title}</h2>
+
+                          <h2 className="font-medium">{t("when")}</h2>
```

Comment:
I don't think having 3 consecutive `h2` tags would be semantically correct.

Reply:
ok what should i use then?

Reply:
Leave it as it was please 🙏🏽 the original issue only mentioned button alignment

Reply:
replaced it with div like it is done on /success page.
Original issue was about button alligment but new design was to be used for this page.